### ISSUE TITLE
Migrating to more recnet material-dialogs?

### ISSUE THREAD

**Issue by idl0r** [sreichholf/dreamDroid#146]
Hi,

wouldn't it make sense to migrate to a more recent material-dialogs, esp. for better night theming and stuff?
0.9.6.0 is long EOL
It would require some changes at least but I am not yet sure how much work it would be in the end.
Using kotlin within a java project seems to be totally fine.

**Comment by sreichholf** [comment]
Yes, I've actually starting updating to Material3 (including the removal of some outdated libs and stuff).

**Comment by sreichholf** [comment]
Closing: obsolete on the 2.0 rewrite (main).
